x-pack/plugin/security/build.gradle

```diff
@@ -454,7 +449,13 @@ tasks.named("thirdPartyAudit").configure {
           'javax.xml.bind.JAXBException',
           'javax.xml.bind.Unmarshaller',
           'javax.xml.bind.UnmarshallerHandler',
-          // Optional dependencies of oauth2-oidc-sdk
+          // Optional dependency of oauth2-oidc-sdk that we don't need since we do not support AES-SIV for JWE
+          'org.cryptomator.siv.SivMode',
+          // Optional dependency of nimbus-jose-jwt for handling Ed25519 signatures and ECDH with X25519 (RFC 8037)
+          'com.google.crypto.tink.subtle.Ed25519Sign',
+          'com.google.crypto.tink.subtle.Ed25519Sign$KeyPair',
+          'com.google.crypto.tink.subtle.Ed25519Verify',
+          'com.google.crypto.tink.subtle.X25519',
```

Comment:
Because http://docs.groovy-lang.org/docs/groovy-2.4.0/html/api/org/codehaus/groovy/runtime/ArrayUtil.html `createArray()` has a max size of args hard-coded to 250 and we hit that limit some months ago.

Reply:
wow ... I didn't know Groovy does that ...